config/overlays/kubeflow/patches/web-app-sidecar.yaml

```diff
@@ -0,0 +1,10 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: models-web-app
+  namespace: kfserving-system
```

Comment:
I think the namespace is[ kustomized](https://github.com/kubeflow/kfserving/blob/master/config/overlays/kubeflow/kustomization.yaml#L5) so we do not need to specify here

Reply:
Removing the namespace from that line would result in
```
Error: no matches for OriginalId apps_v1_Deployment|~X|models-web-app; no matches for CurrentId apps_v1_Deployment|~X|models-web-app; failed to find unique target for patch apps_v1_Deployment|models-web-app
```
since I specifically set the `namespace: kfserving-system` to all the [base/default yamls](https://github.com/kubeflow/kfserving/tree/df64bbc5be6c07f418e0d1d7ce34c16c8d89887e/config/web-app) of the app.

To achieve what you describe I would have to remove the explicit mention of `namespace: kfserving-system` from all the web app's base/default yamls, introduced in this PR.

We slightly went over this with @pvaneck as well https://github.com/kubeflow/kfserving/pull/1512#discussion_r604501150 and in the end decided to stick with using the `namespace: kfserving-system` in the base yamls since this is what the other components are doing as well [ [controller](https://github.com/kubeflow/kfserving/blob/master/config/manager/manager.yaml#L5), [cert-manager](https://github.com/kubeflow/kfserving/blob/master/config/certmanager/certificate.yaml#L7), [configmap](https://github.com/kubeflow/kfserving/blob/master/config/configmap/inferenceservice.yaml#L5), [current sts patch](https://github.com/kubeflow/kfserving/blob/7fc4bbbbd598762536bc9b2965786b7cb310746b/config/overlays/kubeflow/statefulset-patch.yaml#L5) etc ].